### DISCUSSION TITLE
[V5] Possible to run script/js file with build command?

### DISCUSSION THREAD

**Discussion by 3rownz** [adonisjs/core#2509] [Help]
I need to generate a json file from javascript variables. Anyone know if its possible to run a js file during a build, using the `node ace build --production` command?

**Comment by thetutlage** [comment]
Why not create a custom ace command? And then run it alongside the `build` command.

```sh
node ace build --production && node ace <your-command-name>
```

Or better register it as a script in `package.json` file.
